lib/vsc/utils/generaloption.py

```diff
@@ -432,9 +432,10 @@ class ExtOptionParser(OptionParser):
     VALUES_CLASS = Values
     DESCRIPTION_DOCSTRING = False
 
-    ALLOW_OPTION_AS_VALUE = False # exact match for option as value
-    ALLOW_DASH_AS_VALUE = False # any value starting with a '-'
-    ALLOW_TYPO_AS_VALUE = True # value with similarity score from difflib.get_close_matches
+    ALLOW_OPTION_NAME_AS_VALUE = False  # exact match for option name (without the '-') as value
```

Comment:
no it's not, using `-base` as a value is also an issue if `--base` is an option, see tests